features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html

```diff
@@ -89,6 +89,7 @@
         </div>
     </div>
 
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
```

Comment:
why do we need this? 
Also, This could cause issues when the server is used offline.

Reply:
semantic ui script used in this page depends on jquery. We need to open this page on a htmlunit web browser for automating OIDC conformance testing. that browser throws a jquery not defined error.

Reply:
Can we add both `jQuery` and `semantic` libraries into the repo and use without using a cdn? This way it will work on both offline and online scenarios.